Runtime/Core/ApiGatewayManagement/AmazonGameLiftClientWrapper.cs

```diff
@@ -42,17 +57,191 @@ public async Task<SearchGameSessionsResponse> SearchGameSessions(SearchGameSessi
             return await _amazonGameLiftClient.SearchGameSessionsAsync(request);
         }
 
-        public static IAmazonGameLift Create(string localEndpoint)
+        public async Task<DescribeGameSessionsResponse> DescribeGameSessions(DescribeGameSessionsRequest request)
+        {
+            return await _amazonGameLiftClient.DescribeGameSessionsAsync(request);
+        }
+
+        private IAmazonGameLift Create()
+        {
+            var credentials = SetupCredentials();
+            return new AmazonGameLiftClient(credentials.AccessKey, credentials.SecretKey);
+        }
+        #endregion
+        
+        /// <summary>
+        /// Server region is code dedicated to Amazon GameLift SDK and AWS SDK calls made by the game server. All of these calls will be done via UI Elements or on Startup. 
+        /// </summary>
+        #region Server
+        
+        public AmazonGameLiftClientWrapper()
+        {
+            _gameLiftClientSettings = Resources.FindObjectsOfTypeAll<GameLiftClientSettings>()[0];
```

Comment:
Could you help me understand what's happening here? Is Unity provided construct? Doesn't seem like a generalizable way to pull client settings, what happens if there are multiple such settings in the future?

Reply:
As the new UI is also being worked on, I have thought about this implementation more. I think that the settings for this will not be able to be duplicated, as the UI will be targeting one single settings object for the user to edit. When this is being implemented into the UI, I will see if I can come up with changes to make it more generalizable incase I am wrong.

Reply:
Could you point me to how these settings were earlier being fetched from Unity? Want to see what the before vs after. I get your point but my gut feeling says that there should be a better way to store and access these from Unity? We can discuss offline if that helps make this discussion easier

Reply:
So now that you point this out I have found the version of settings storing you were doing before using CoreApi and storing it within a GameLiftSettings.Yaml file. The Gameliftclientsettings scriptable object we have was just a user facing object that they would interact with but we would save that data in the file using core api. When I am working on the UI, I will save their data like how was originally done and change the code to call these settings instead. 
Thanks for pointing this out.

Reply:
Thank you for following up on this investigation. If this will not be completed as part of this commit, please add TODO comments with the relevant task so this doesn't fall over.